src/Graphics/OpenGLContext/opengl_ContextImpl.cpp

```diff
@@ -527,6 +527,11 @@ bool ContextImpl::isSupported(graphics::SpecialFeatures _feature) const
 	return false;
 }
 
+u32 ContextImpl::getMaxMSAALevel()
+{
+	return FunctionWrapper::maxMSAALevel();
```

Comment:
Why do we need special FunctionWrapper::maxMSAALevel() here?
Why not just call glGetIntegerv directly:
```
s32 ContextImpl::getMaxMSAALevel() const
{
	GLint maxMSAALevel;
	glGetIntegerv(GL_MAX_SAMPLES, &maxMSAALevel);
	return maxMSAALevel;
}

```